### ISSUE TITLE
ability to hide all self text posts?

### ISSUE THREAD

**Issue by jawz101** [ccrama/Slide#3161]
I'm not a fan of showerthoughts, iama, ama, tifu, eli5, .. really any self posts in any sub.  Could there be a way to simply filter out tall self. content?

Between that and blocking YouTube, redd.it, imgur, Twitter, giphycat, and Wikipedia links reddit turns into just good stories.

**Comment by Alexendoo** [comment]
I don't believe there is, continuing blocking the subreddits might be the easiest way. Or perhaps to opposite of creating your own subscriptions

**Comment by jawz101** [comment]
I think I found out where it's done.  For each subreddit, click the 3 dots & got to Subreddit content filter.  Under there you can filter out all selftext posts
